File: primer_trimestre/poo/java/proyectos_java/ejercicio_3_java/src/secondLaw.java
public class secondLaw extends NewtonLaw {
    public secondLaw(double mass, double acceleration, double force) {
        super(mass, acceleration, force);
    }
    @Override
    public String calculate() {
        double mass = getMass();
        double acceleration = getAcceleration();
        double force = mass*acceleration;
        return "The force is: " + force + " N";
    }
}

File: primer_trimestre/poo/php/ejercicios_php/third/libreria/secondLaw.php
<?php
require_once 'NewtonLaw.php';
class SecondLaw extends NewtonLaw {
    public function __construct(float $mass, float $acceleration, float $force) {
        parent::__construct($mass, $acceleration, $force);
    }

    public function calculate(): string {
        $mass = $this->getMass();
        $acceleration = $this->getAcceleration();
        $force = $mass * $acceleration;
        return "The force is: " . $force . " N";
    }
}
